ouroboros-network-protocols/test/Test/AnchoredFragment.hs

```diff
@@ -295,21 +295,31 @@ prop_intersect :: TestAnchoredFragmentFork -> Property
 prop_intersect (TestAnchoredFragmentFork origP1 origP2 c1 c2) =
```

Comment:
Could you move these tests to `ouroboros-network-api`?

Reply:
We just talked about this, and this doesn't have to happen in this PR; @coot also has some general restructuring plans.